app/views/dashboard/_motd.html.erb

```diff
@@ -1,18 +1 @@
-<h3>Message of the Day</h3>
-
-<% if motd.messages.each do |m| %>
-
-  <hr>
-  <div class="motd">
-    <h4 class="motd_title"><%= m.date %> - <%= m.title %></h4>
-    <div class="motd_body"><p><%=raw markdown( m.body ) %></p></div>
-  </div>
-
-<% end.empty? %>
-
-  <hr>
-  <div class="motd">
-    <h4 class="motd_title">There are no new messages.</h4>
-  </div>
-
-<% end %>
+<%= render motd_file %>
```

Comment:
Can this file be deleted? Is it a duplicate of another? Is this used anywhere else?